fsspec/implementations/smb.py

```diff
@@ -199,7 +208,13 @@ def _open(
         if "w" in mode and autocommit is False:
             temp = _as_temp_path(self.host, path, self.temppath)
             return SMBFileOpener(wpath, temp, mode, block_size=bls, **kwargs)
-        return smbclient.open_file(wpath, mode, buffering=bls, **kwargs)
+        return smbclient.open_file(
```

Comment:
I suggest that kwargs should still allow for overriding the default value for those that use fs.open directly. One way:
```python
    share_access = kwargs.pop("share_access", self.share_access)
    return smbclient.open_file(.., share_access=share_access, **kwargs)
```
(or you can include it as an explicit kwarg 
```python
_open(.., share_access=None, **kwargs):
    share_access = share_access or self.share_access
    ...
```

Reply:
Right, good idea. I've opted for the first option, since this method overrides from AbstractFileSystem, I prefer not to change its signature.